Create conary.yaml

Pierre-Gronau-ndaal opened the pull request.

Pierre-Gronau-ndaal committed:
Create conary.yaml

tclahr commented:
I am not familiar with conary. Is it used by any linux flavor? I saw that the project is no longer supported or maintained (https://github.com/sassoftware/conary) in Github.

Pierre-Gronau-ndaal commented:
Conary is used by SAS Linux and Foresight Linux (https://www.foresightlinux.org/)

even if something is actual this doesn´t mean that a forensic tool is only looking for bleeding age distros - just my 5 cents

tclahr committed:
Merge branch 'develop' into patch-18

tclahr committed:
artif: new artifact

Added collection of the list of installed packages managed by the Conary package manager.
